src/shipit_static_analysis/shipit_static_analysis/lint.py

```diff
@@ -192,6 +192,15 @@ def find_issues(self, path, revision):
             path
         ]
         returncode, output, error = run(' '.join(command), cwd=settings.repo_dir)
+
+        # Dump raw mozlint output as a Taskcluster artifact (for debugging)
+        output_path = os.path.join(
+            settings.taskcluster_results_dir,
+            '{}-mozlint.txt'.format(revision),
```

Comment:
Same thing here, the generated url would be dangerous, and long.

Reply:
:heavy_check_mark: